src/viam/app/app_client.py

```diff
@@ -568,6 +620,24 @@ async def update_organization_invite_authorizations(
         Note that an invite can only have one authorization at each resource (e.g., organization, location, robot, etc.) level and must have
         at least one authorization overall.
 
+        ::
+
+            from viam.proto.app import Authorization
+
+            authorization_to_add = Authorization(
+                authorization_type="some type of auth",
+                authorization_id="identifier",
+                resource_type="abc",
+                resource_id="resource-identifier123",
+                identity_id="id12345",
+                organization_id="org_id_123"
+            )
+
+            update_invite = await cloud.update_organization_invite_authorizations(
+                email="notarealemail@viam.com",
+                remove_authorizations=[authorization_to_add]
```

Comment:
good catch - thanks! 
```suggestion
                add_authorizations =[authorization_to_add]
```